### ISSUE TITLE
Verse Indicators

### ISSUE THREAD

**Issue by TheSecEng** [caleb531/youversion-suggest-alfred#12]
**What**
Verse indicators when copying multiple verse, is that possible?

**Comment by calebevans-suitecentric** [comment]
@TheSecEng It's a setting under `yvset`:

```
yvset versenumbers yes
```

This will add the verse numbers to the copied output of any number of verses. Is this what you meant?

**Comment by caleb531** [comment]
@TheSecEng It's a setting under yvset:

yvset versenumbers yes
This will add the verse numbers to the copied output of any number of verses. Is this what you meant?

**Comment by TheSecEng** [comment]
Yes, however I am getting some odd output. See how there are random pound signs?

`yvfilter Matthew 6 13-19`

```
Matthew 6:13-19 (NKJV)

13 # And do not lead us into temptation,
But # deliver us from the evil one.
For Yours is the kingdom and the power and the glory forever. Amen.

14 # “For if you forgive men their trespasses, your heavenly Father will also forgive you. 15 But # if you do not forgive men their trespasses, neither will your Father forgive your trespasses.

16 “Moreover, # when you fast, do not be like the hypocrites, with a sad countenance. For they disfigure their faces that they may appear to men to be fasting. Assuredly, I say to you, they have their reward. 17 But you, when you fast, # anoint your head and wash your face, 18 so that you do not appear to men to be fasting, but to your Father who is in the secret place; and your Father who sees in secret will reward you openly.

19 # “Do not lay up for yourselves treasures on earth, where moth and rust destroy and where thieves break in and steal;
```

**Comment by caleb531** [comment]
@TheSecEng Are the pound signs (#) still present if you disable verse numbering?

```
yvset versenumbers no
```

At any rate, this is probably a bug, and something I will aim to fix over the next few days.

**Comment by TheSecEng** [comment]
The `#` are only there when specifying it to include verse numbers.

Again, thanks for this workflow. It is insanely good.

**Comment by caleb531** [comment]
@TheSecEng Good news! I have just released YouVersion Suggest v11.3.1 with the precise fix. Please try it out and let me know how it works for you.

https://github.com/caleb531/youversion-suggest-alfred/releases/tag/v11.3.1

Blessings,
Caleb

**Comment by TheSecEng** [comment]
> @TheSecEng Good news! I have just released YouVersion Suggest v11.3.1 with the precise fix. Please try it out and let me know how it works for you.
> 
> https://github.com/caleb531/youversion-suggest-alfred/releases/tag/v11.3.1
> 
> Blessings,
> Caleb

Awesome ! Will try it tonight.

**Comment by TheSecEng** [comment]
Perfect! Thanks
